src/components/box/box.spec.js

```diff
@@ -1,16 +1,74 @@
 import React from "react";
-
+import { mount } from "enzyme";
 import {
   testStyledSystemSpacing,
   testStyledSystemColor,
   testStyledSystemLayout,
   testStyledSystemFlexBox,
+  assertStyleMatch,
 } from "../../__spec_helper__/test-utils";
 import Box from "./box.component";
+import BaseTheme from "../../style/themes/base";
 
 describe("Box", () => {
   testStyledSystemSpacing((props) => <Box {...props} />);
   testStyledSystemColor((props) => <Box {...props} />);
   testStyledSystemLayout((props) => <Box {...props} />);
   testStyledSystemFlexBox((props) => <Box {...props} />);
+
+  it("has styles applied when scrollVariant is set to light", () => {
```

Comment:
These tests could also be simplified by using `it.each`

```jsx
it.each(['light', 'dark], "has styles applied when scrollVariant is set to %s", (scrollVariant) => {
  const wrapper = mount(<Box scrollVariant={scrollVariant} />);
  // ...
   assertStyleMatch(
      {
        backgroundColor: BaseTheme.scrollbar[scrollVariant]track,
      },
      wrapper,
      { modifier: "::-webkit-scrollbar-track" }
    );
  // ...
})
```

Reply:
Yeah this is better. Ive changed it over